app/uk/gov/hmrc/essttpstubs/repo/EligibilityRepo.scala

```diff
@@ -46,18 +47,20 @@ final class EligibilityRepo @Inject() (
   def insertEligibilityData(eligibilityEntry: EligibilityEntry): Future[UpdateResult] =
 
     collection.replaceOne(
-      Filters.and(eligibilityEntry.eligibilityCheckResult.identification.map(_.idValue.value).map(taxIdFilter): _*),
+      Filters.and(eligibilityEntry.eligibilityCheckResult.identification.map(_.idValue.value)
```

Comment:
could you not make use of the new `taxIdFilter` here?

Reply:
oh yes